internals/core_models.py

```diff
@@ -66,11 +66,14 @@ class FeatureEntry(ndb.Model):  # Copy from Feature
   flag_name = ndb.StringProperty()
   ongoing_constraints = ndb.TextProperty()
 
-  # Gate: Adoption
+  # Topic: Adoption (reviewed by API Owners.  Auto-approved gate later?)
   motivation = ndb.TextProperty()
   devtrial_instructions = ndb.TextProperty()
   activation_risks = ndb.TextProperty()
   measurement = ndb.TextProperty()
+  availability_expectation = ndb.TextProperty()
+  adoption_expectation = ndb.TextProperty()
+  adoption_plan = ndb.TextProperty()
```

Comment:
Is there ever a scenario where multiple adoption gates would exist for the same feature? If so, it might be useful for these fields to live on the Stage or Gate kinds.

Reply:
Also, I was going to suggest that maybe we add comments here to explain the usage of each of these fields, but I think that would be a useful for all of our fields. I'll write up an issue as a possible enhancement.

Reply:
> Is there ever a scenario where multiple adoption gates would exist for the same feature? If so, it might be useful for these fields to live on the Stage or Gate kinds.

Good question, but in this case I think that the feature owner would revise these fields rather than have separate copies of them.  Even a feature with multiple launches is still one feature that gets adopted by web developers.